enhancements/installer/image-based-installer.md

```diff
@@ -254,27 +254,38 @@ N/A
 
 ## Version Skew Strategy
 
-### Configuration ISO version and Seed OCI Image version
-
-The IBI configuration ISO content version should be compatible with the seed
-OCI image version contained in the IBI installation ISO. The Lifecycle Agent
-Operator, which is contained in the installation ISO, is responsible for
-ensuring the compatibility of those two at the time of the image-based
-installation. The configuration version does not depend on the OpenShift
-versions used to generate either one of the IBI artifacts (i.e. configuration
-ISO and installation ISO). In the case of incompatibility between the
-configuration version and the seed OCI image version, the image-based
+### Configuration ISO and Seed OCI Image
+
+The IBI configuration ISO contains the [cluster configuration file](https://github.com/openshift-kni/lifecycle-agent/blob/release-4.16/api/seedreconfig/seedreconfig.go).
+The component responsible for parsing and using the latter is part of the seed
+OCI image, which in turn is contained in the IBI installation ISO. That
+component validates its compatibility with the cluster configuration file [version](https://github.com/openshift-kni/lifecycle-agent/blob/release-4.16/api/seedreconfig/seedreconfig.go#L6)
+during the image-based installation. In the case of incompatibility between the
+cluster configuration version and the seed OCI image version, the image-based
 installation will fail with the respective error message.
 
-### Installation ISO RHCOS ISO version and Seed OCI Image version
+### RHCOS ISO and Seed OCI Image
 
 The RHCOS base ISO, which is contained in the IBI installation ISO and derived
-from the OpenShift release payload, has no strict requirements to be tied to the
-seed OCI image OpenShift version. The features and configuration of the
-underlying tools required to successfully complete an image-based
-installation, e.g. Podman, cri-o, SELinux, ostree, do not frequently change
-between OpenShift versions. The risk of version skew is small and in such case
-the image-based installation will fail with the respective error message.
+from the OpenShift release image, has currently no strict requirements to be
+tied to the seed OCI image OpenShift version. The features and configuration of
+the underlying tools required to successfully complete an image-based
+installation, e.g. Podman, SELinux, ostree, do not frequently change between
```

Comment:
Given previous answers, is it that case that as far as the installation image is concerned, the seed image is a just a container image and RHCOS doesn't care what's in it, or is that going too far?
It would seem like the main worry here is that the code in the installation image needs to know something about how the seed image is organised, and you might need a versioning mechanism to make sure you are not permanently locked in to what the seed image looks like today.

Reply:
Actually there is nothing installation should know about seed rather than version in order to validate that user knows what he installs. 
All other actions are independent as we just use ostree in order to write seed image to disk